userspace/sysdig/csysdig.cpp

```diff
@@ -1012,7 +1039,7 @@ sysdig_init_res csysdig_init(int argc, char **argv)
 
 			if(output_type == sinsp_table::OT_JSON)
 			{
-				printf("]}\n");
+				//printf("]}\n");
 				//printf("%c", EOF);
 			}
```

Comment:
should this be removed?

Reply:
based on what I'm seeing, that line produces malformed json, and that's why I commented it. However, I'm worried it's required by some other tools (sysdig inspect in particular) and if by removing it those tools will break. That's why I just commented it.

Would CI/CD catch if we break inspect?